app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PagesController < ApplicationController
+  def home
+    @products = Product.all
```

Comment:
Using this approach, if a user selects for example both categories pizza & dessert, the query will look like this:
```
Product.all.where(category: :pizza).where(category: :dessert)
# which in SQL will be
SELECT COUNT(*) FROM "products"  WHERE "products"."category" = 'pizza' AND "products"."category" = 'dessert'
```

So this will mean that it will only show results that are both pizza & dessert.

Reply:
As far as I can see, these fields are enums, so a product can only 1 at a time. Especially for the vegetarian/non-vegetarian part, this will not work. For example, you can try doing in Rails console:
```ruby
Product.all.where(vegetarian: :non_vegetarian).where(vegetarian: :vegetarian).count
```
and the result will always be 0.

Reply:
So in case you intended to return products that are either pizza or dessert when selecting both pizza & dessert, this is not working. If you only want to select 1 category and return for that, it is fine to leave it like this